src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Component/Permissions/PermissionExtensionConfigure.cs

```diff
@@ -30,7 +30,7 @@ protected override List<UniqueModel> ExpansionWrapperUniqueValue
 
     protected override async Task RoleUnionTeamPermissionValueChangedAsync()
     {
-        ExtensionValue = ExtensionValue.Where(e=>e.Effect||(e.Effect==false&&RoleUnionTeamPermission.Contains(e.PermissionId))).ToList();
+        ExtensionValue = ExtensionValue.Where(e => e.Effect || (e.Effect == false && RoleUnionTeamPermission.Contains(e.PermissionId))).ToList();
```

Comment:
```suggestion
        ExtensionValue = ExtensionValue.Where(e => e.Effect || RoleUnionTeamPermission.Contains(e.PermissionId)).ToList();
```
I don't think the removed code is nessary